hubot-web-screenshot/README.md

```diff
@@ -0,0 +1,17 @@
+# hubot-web-screenshot
+
+Takes a picture of a website and shows it to you. Currently works with slack.
+
+# Setup
+
+Add hubot app to slack and create an API key. Set the API key as an environment variable like so
+
+```
+HUBOT_SLACK_TOKEN=xoxb-xxcjqneigb3415-452130989397-314i13jtn24jgn4534
```

Comment:
Ooops? :)

Reply:
The token isn't actually real. It looks like the same format as a slack token but i made it up by dragging my face across the keyboard a few times.